src/main/kotlin/no/nav/klage/oppgave/service/TilgangService.kt

```diff
@@ -28,20 +29,33 @@ class TilgangService(
 
     fun verifyInnloggetSaksbehandlersSkrivetilgang(klagebehandling: Klagebehandling) {
```

Comment:
Trenger vi denne?

Reply:
Vi bruker den i en kjede fra `ferdigstillKlagebehandling` nå. Tipper den kan fjernes på sikt, men det blir lettere når vi tar for oss det endepunktet, tenker jeg.

Reply:
Kan ikke den nye metoden (behandling) brukes like gjerne? De ser helt like ut?